HACKING.md

```diff
@@ -450,45 +513,50 @@ with `gunzip` and placed into `~/.spread/qemu` as above.
 
 #### Running spread with QEMU
 
-Finally, you can run the spread tests for Ubuntu 16.04 LTS 64-bit with:
+Finally, you can run the spread tests for Ubuntu 18.04 LTS 64-bit with:
 
-    $ spread -v qemu:ubuntu-16.04-64
+    $ spread -v qemu:ubuntu-18.04-64
 
-To run for a different system, replace `ubuntu-16.04-64` with a different system
-name.
+>To run for a different system, replace `ubuntu-18.04-64` with a different system
+name, which should be a basename of the [built](#building-spread-vm-images) or 
+[downloaded](#downloading-spread-vm-images) Ubuntu image file.
 
 For quick reuse you can use:
 
-    $ spread -reuse qemu:ubuntu-16.04-64
+    $ spread -reuse qemu:ubuntu-18.04-64
 
 It will print how to reuse the systems. Make sure to use
 `export REUSE_PROJECT=1` in your environment too.
 
 #### Running UC20 spread with QEMU
 
+<!-- TODO: (B) Is it still working this way? -->
+
 Ubuntu Core 20 on amd64 has a requirement to use UEFI, so there are a few 
-additional steps needed to run spread with the ubuntu-core-20-64 systems locally
+additional steps needed to run spread with the `ubuntu-core-20-64` systems locally
 using QEMU. For one, upstream spread currently does not support specifying what
 kind of BIOS to use with the VM, so you have to build spread from this PR:
 https://github.com/snapcore/spread/pull/95, and then use the environment 
 variable `SPREAD_QEMU_BIOS` to specify an UEFI BIOS to use with the VM, for 
-example the one from the OVMF package. To get OVMF on Ubuntu, you can just 
-install the `ovmf` package via `apt`. After installing OVMF, you can then run 
-spread like so:
+example the one from the OVMF package.  
+
+>To get OVMF on Ubuntu, you can just install the `ovmf` package via `apt`.  
+
+After installing OVMF, you can then run spread like so:
 
     $ SPREAD_QEMU_BIOS=/usr/share/OVMF/OVMF_CODE.fd spread -v qemu:ubuntu-core-20-64
 
 This will enable testing UC20 with the spread, albeit without secure boot 
 support. None of the native UC20 tests currently require secure boot however, 
-all tests around secure boot are nested, see the section below about running the
-nested tests.
+all tests around secure boot are nested, see the section below about 
+[running the nested tests](#running-nested-tests).
 
 Also, due to the in-flux state of spread support for booting UEFI VM's like 
-this, you can test ubuntu-core-20-64 only by themselves and not with any other
+this, you can test `ubuntu-core-20-64` only by themselves and not with any other
 system concurrently since the environment variable is global for all systems in
 the spread run. This will be fixed in a future release of spread.
 
-### Testing snapd
+### Testing `snapd` daemon
```

Comment:
This looks off to me, it should be "Testing the snapd daemon" or "Testing snapd" (the original version). IMO I'd go with the original

Reply:
The thing with _the_ is my weak point. :-)

I still have a question. From what I understood this section describes the tests that cover only `snapd` daemon. The entire package is named `snapd`. Other tests here also test `snapd`. 

Doesn't it require clarification?

Reply:
I prefer "Testing the snapd daemon", since these tests cover only daemon, whereas all tests referred by this document test `snapd`. I'll check this "_the_" thing with Graham.